src/app/helptext/task-calendar/replication/replication.ts

```diff
@@ -2,234 +2,243 @@ import { T } from '../../../translate-marker';
+ include in the replication with the periodic snapshot schedule. Enter \
+ the <a href="https://www.freebsd.org/cgi/man.cgi?query=strftime" \
+ target="_blank">strftime(3)</a> strings that match the snapshots to \
+ include in the replication.<br><br> \
+ When a periodic snapshot is not linked to the replication, enter the \
+ naming schema for manually created snapshots. Has the same <i>%Y</i>, \
+ <i>%m</i>, <i>%d</i>, <i>%H</i>, and <i>%M</i> string requirements as \
+ the <b>Naming Schema</b> in a \
+ <a href="--docurl--/tasks.html#periodic-snapshot-tasks" \
  target="_blank">periodic snapshot task.'),
 
     auto_placeholder: T('Run Automatically'),
-    auto_tooltip: T('Set to either start this replication task\
- immediately after the linked periodic snapshot task completes or see\
+    auto_tooltip: T('Set to either start this replication task \
+ immediately after the linked periodic snapshot task completes or see \
  options to create a separate <b>Schedule</b> for this replication.'),
 
     schedule_placeholder: T('Schedule'),
-    schedule_tooltip: T('Define specific times to start snapshotting the\
- <b>Source Datasets</b>. Disables running the replication immediately\
- after the periodic snapshot task. Adds the <b>Begin</b> and <b>End</b>\
- fields.'),
+    schedule_tooltip: T('Define a specific time to start the replication \
+ task.'),
 
-    schedule_picker_tooltip: T('Select a preset schedule or choose\
- <i>Custom</i> to use the advanced scheduler. Spaces are not allowed in\
+    schedule_picker_tooltip: T('Select a preset schedule or choose \
+ <i>Custom</i> to use the advanced scheduler. Spaces are not allowed in \
  <b>Minutes</b>, <b>Hours</b>, or <b>Days</b> of the custom scheduler.'),
 
     schedule_begin_placeholder: T('Begin'),
     schedule_begin_tooltip: T('Start time for the replication task.'),
 
     schedule_end_placeholder: T('End'),
-    schedule_end_tooltip: T('End time for the replication task. A\
+    schedule_end_tooltip: T('End time for the replication task. A \
  replication that is in progress can continue to run past this time.'),
 
     restrict_schedule_placeholder: T('Snapshot Replication Schedule'),
-    restrict_schedule_tooltip: T('Schedule which periodic snapshots are\
- replicated. All snapshots are replicated by default. To choose which\
- snapshots are replicated, set the checkbox and select a schedule from\
- the drop-down menu. For example, the periodic snapshot task takes a\
- snapshot every hour, but only every other snapshot is needed for\
- replication. The scheduler is set to even hours and only snapshots\
+    restrict_schedule_tooltip: T('Schedule which periodic snapshots are \
+ replicated. All snapshots are replicated by default. To choose which \
+ snapshots are replicated, set the checkbox and select a schedule from \
+ the drop-down menu. For example, the periodic snapshot task takes a \
+ snapshot every hour, but only every other snapshot is needed for \
```

Comment:
This seems to be a specific example, but reads like a statement of fact: "only every other snapshot is needed". Reword to clarify that this is an example.  "As an example, consider a system which takes a snapshot every hour, but only replication of every other snapshot is desired."